CHANGELOG.rst

```diff
@@ -41,6 +50,7 @@ Fixed
 
 Changed
 =======
+
```

Comment:
Nit: Stray line addition. (I don't have an opinion on the formatting, but I'm guessing it was unintentional, from a merge resolution).